src/config/authInfoConfig.ts

```diff
@@ -5,7 +5,7 @@
  * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 
-import { AuthFields } from '../org';
+import { AuthFields } from '../org/authInfo';
```

Comment:
there'll be a lot of diffs that look like this as a result of removing barrel files